rx-java-2/gradle.properties

```diff
@@ -0,0 +1 @@
+moduleArtifactId=rx-java-2
```

Comment:
Please prefix the artifact id with `c3media-` as it was before. The name of the generated jar file should always state the name of the library such as `c3media-rx-java-2-0.1.0-SNAPSHOT.jar`. As a result you also have to update the dependency statement to `compile "info.metadude.kotlin.library.c3media:c3media-rx-java-2:$version"`.

Reply:
Fixed in #2.